TodoApp/Src/TodoApp.Dependency/Resolvers/DependencyResolver.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http.Formatting;
+using System.Web.Http.Controllers;
+using System.Web.Http.Dependencies;
+using System.Web.Http.Dispatcher;
+using System.Web.Http.ExceptionHandling;
+using System.Web.Http.Hosting;
+using System.Web.Http.Metadata;
+using System.Web.Http.Validation;
+using TodoApp.Contracts.Containers;
+using TodoApp.Dependency.Exceptions;
+
+namespace TodoApp.Dependency.Resolvers
+{
+    internal class DependencyResolver : IDependencyResolver
+    {
+        private static readonly Type[] ExcludedTypes =
+        {
+            typeof(IHostBufferPolicySelector),
+            typeof(IHttpControllerSelector),
+            typeof(IHttpControllerActivator),
+            typeof(IHttpActionSelector),
+            typeof(IHttpActionInvoker),
+            typeof(IContentNegotiator),
+            typeof(IExceptionHandler),
+            typeof(ModelMetadataProvider),
+            typeof(IModelValidatorCache)
+        };
```

Comment:
I'm thinking whether these are not types that would be worth specifying in `Api` assembly. Once you solve the [route naming problem](https://github.com/pthutta/kentico-onboarding-cs/pull/3/files#diff-05426ec05dc6094517d16a3b01be4feaR9), the solution to this puzzle will be quite obvious for sure. Whether implement it and move responsibility for customizations to the `Api` where they originate or leave them here where they would be implemented is just something to consider… (I guess I preffer current solution, however, I would like to evalue the option as well)